content/en/docs/installation/guide/redhat-marketplace.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Install from Red Hat Marketplace
+linkTitle: Install from Red Hat Marketplace
+weight: 2
+aliases: 
+  - /spinnaker/openshift-container-platform/red-hat-marketplace
+  - /spinnaker/openshift-container-platform/red-hat-marketplace
+---
+
+{{% alert title="Note" %}}This document is intended for users who have purchased the Armory Red Hat Marketplace offering. It will not work if you have not purchased the Armory Operator.
+
+Please contact [Armory](mailto:hello@armory.io) if you're interested in a Red Hat Marketplace Private Offer.{{% /alert %}}
+
+## Overview
+
+The Armory Operator is a Kubernetes Operator for Spinnaker<sup>TM</sup> that makes it easier to install, deploy, and upgrade Spinnaker. 
+
+## What is the Red Hat Marketplace?
+
+The Red Hat Marketplace is available directly from your OpenShift web console and provides an open cloud catalog that makes it easier to discover and access certified software for container-based environments in public clouds. With automated deployment, software is immediately available to deploy on any Red Hat OpenShift cluster, providing a fast, integrated experience. Discover and buy certified software, and quickly deploy. Access open source and proprietary software, with responsive support, streamlined billing and contracting, simplified governance, and single-dashboard visibility across clouds. Built in partnership by Red Hat and IBM, this marketplace helps organizations deliver enterprise software and improve workload portability.
+
+## Before you begin
+
+{{% alert title="Note" %}}You must have the cluster administrator role to install the operators from the Red Hat Marketplace.{{% /alert %}}
+ 
+1. These instructions assume you already have a Kubernetes cluster available in OpenShift Container Platform v4.4+
+2. Browse to the [Red Hat Marketplace](https://marketplace.redhat.com/en-us?_ga=2.94000664.441560370.1604955054-123808014.1597373423) and log in or create a new account.
+3. Follow the [instructions](https://marketplace.redhat.com/en-us/documentation/clusters?&_ga=2.94000664.441560370.1604955054-123808014.1597373423#register-openshift-cluster-with-red-hat-marketplace) to register your OpenShift cluster with the Red Hat Marketplace.
+4. When prompted `Would you like to go back to the Red Hat Marketplace now? [Y/n]`, type `Y` and the Red Hat Marketplace page is opened in your browser.
+5. Click <b>My software</b> >  <b>Visit the Marketplace.</b>
```

Comment:
I think `<strong>` is preferred these days over `<b>`; are we getting these instructions straight from RedHat?

Reply:
I'll be replacing the HTML with Markdown. :)

Reply:
Thank you @aimeeu

Reply:
@dogonthehorizon kind of, these instructions are implicit here https://marketplace.redhat.com/en-us/documentation/getting-started